src/scripts/assume-role-with-web-identity.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+PARAM_ROLE_SESSION_NAME=$(eval echo "${PARAM_ROLE_SESSION_NAME}")
```

Comment:
Why not `PARAM_ROLE_SESSION_NAME=${!PARAM_ROLE_SESSION_NAME}` ?

Bash supports expansion from variable name natively: https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html#Shell-Parameter-Expansion

Reply:
@f0o, 

By default, the role session name uses the environment variable `${CIRCLE_JOB}` as a default value.

If you just do `${!PARAM_ROLE_SESSION_NAME}` you'd get `CircleCi-${CIRCLE_JOB}`.  I've tried to using `CircleCi-${!CIRCLE_JOB}` but that doesn't work either.

Reply:
@brivu so the variable `PARAM_ROLE_SESSION_NAME` is set like `PARAM_ROLE_SESSION_NAME='CircleCI-${CIRCLE_JOB}'` literally without expansion on setting?

Reply:
Hey @f0o, 

This is great my friend. Let me look into this more and give it a shot.  Appreciate your contribution. 

-Brian